.dockerignore

```diff
@@ -0,0 +1,17 @@
+# Deny All
+*
```

Comment:
this is not a traditional approach, but I think I like 👍 
clearly implies we're being discerning about what's being published